lib/thredded/seed_database.rb

```diff
@@ -21,6 +29,13 @@ def self.run(users: 5, topics: 26, posts: (0..20))
       s.create_topics(count: topics)
       s.create_posts(count: posts)
       s.create_private_posts(count: posts)
+    ensure
+      # Re-enable callbacks
+      SKIP_CALLBACKS.each { |(klass, *args)| klass.set_callback(*args) }
```

Comment:
![](http://i.imgur.com/gsU4q93.gif)

"Hello. I am here to make it fancy af up in this bish"